fastlane/Fastfile

```diff
@@ -105,6 +108,63 @@ platform :ios do
     # Remove any previous zipped SDK products
     FileUtils.remove_file("../#{deploy_zip}", force = true)
   end
+
+desc "Sets the SDK version in all files"
+  lane :set_version do
+    UI.important "[ATTENTION] Version number should be MAJOR.MINOR.PATCH, do NOT include the `v` prefix."
+    version = UI.input "Enter version number:"
+
+    # Set MARKETING_VERSION using `versioning` plugin
+    increment_version_number_in_xcodeproj(
+      version_number: version,
+      xcodeproj: project_file_path
+    )
+    UI.message "MARKETING_VERSION updated to #{version}"
+
+    version_bump_podspec(path: podspec_path, version_number: version)
+    UI.message "Podspec version updated to #{version}"
+
+    UI.important "You will now update the Changelog. You will be prompted to add entries for any of the available change types. See https://keepachangelog.com/en/1.0.0/ for format."
+
+    today = DateTime.now.strftime("%Y-%m-%d")
+    changelog_changes = "## [#{version}] - #{today}\n"
+    included_change_type_index = 0 # For keeping track of which change types we include to format empty lines correctly
+
+    change_types.each { |type|
+      input = UI.input "Include entry for `#{type}`? (y/n)"
+      if (input.casecmp?(yes_input))
+
+        if (included_change_type_index > 0)
+          changelog_changes += "\n" # Add an extra line between change type sections
+        end
+
+        included_change_type_index += 1
+        changelog_changes += "### #{type}\n" # Add change type title
+
+        finished = false
+        while(!finished)
+          UI.important "[ATTENTION] Do NOT include the `-` prefix in your entry."
```

Comment:
You can strip any leading `-` characters from the input too (or decide if you need to prefix it if they already have).